azure_functions_worker/dispatcher.py

```diff
@@ -330,7 +334,7 @@ async def _handle__functions_metadata_request(self, request):
         if not os.path.exists(function_path):
             # Fallback to legacy model
             logger.info("%s does not exist. "
-                        "Switching to host indexing.", SCRIPT_FILE_NAME)
+                        "Switching to host indexing.", script_file_name)
```

Comment:
If this is the only place this variable is going to be used, why not add the method invocation inside it? Same for below method?

Reply:
It's used twice in metadata_request but only once in load_request. Changed to method invocation for load_request.